### ISSUE TITLE
PNPM workspace: protocol - wrong version rewrite

### ISSUE THREAD

**Issue by StarpTech** [ghiscoding/lerna-lite#198]
### Describe the bug

Hi, lerna-lite replaced `>=` with `^`. I'm not sure if this is correct.

### Reproduction

1. Create a pnpm workspace with two packages (FOO, SDK) and FOO depending on SDK.

FOO: version 0.80.0, use the version constraint `workspace:>=0.91.0` on SDK
SDK: version 0.91.0

2. Make a change in SDK and commit

Calculate the next version with `lerna publish prerelease --no-verify-access --conventional-commits --create-release github` lerna will bump the SDK version in FOO as follow.

```diff
---    "SDK": "workspace:>=0.91.0",
+++    "SDK": "workspace:^0.91.6-next.1",
```

### Lerna config and logs

<details><summary>lerna.json</summary><p>

```json
<!-- Please paste your `lerna.json` here -->
```
</p></details>

<details><summary>lerna-debug.log</summary><p>

```txt
<!-- If you have a `lerna-debug.log` available, please paste it here -->
<!-- Otherwise, feel free to delete this <details> block -->
```
</p></details>

### Environment Info

```shell
| Executable        | Version |
| ----------------- | ------- |
| `lerna --version` | 1.4.0 |
| `npm --version`   | VERSION |
| `yarn --version`  | VERSION |
| `node --version`  | v16.14.2 |

---
OR simply run `npx lerna info` command
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/lerna-lite/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://github.com/ghiscoding/lerna-lite#readme).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/lerna-lite/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/lerna-lite/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by StarpTech** [comment]
> Question, are you migrating from Lerna? Was that command working in Lerna?

No, first time in that repo https://github.com/wundergraph/wundergraph

> Would you like to take a look and submit a PR? I'm still busy looking at how to deal with the other open PR to update pnpm lock file.

Yes, but it doesn't block me. I'll fall back to `^` for now.


> As a side note and reminder, I'm just a regular developer who created this Lerna-Lite fork from Lerna and I don't fully understand all of Lerna's code, though I'm trying to help the community as much as I can with this fork. Thanks for all the useful feedback you provided already and patience as well

Thanks for the words. Lerna has no support for the workspace protocol. You enabled it for me. You rock! Keep up the good work and don't try to be a clone of Lerna. I like your methodology: Small utility functions for release and versioning.

**Comment by ghiscoding** [comment]
The reason I asked if you came from Lerna is because I'm not sure how Lerna's code does the bump like I said earlier so I'm not sure about your expectation with the flags you used. Could you clarify your problem description, 

```diff
---    "SDK": "workspace:>=0.91.0",
+++    "SDK": "workspace:^0.91.6-next.1",
```
is the red one the expectation and the green one the actual? I'm not sure about the `next.1` part

**Comment by StarpTech** [comment]
That's the change after bumping. Honestly, I'm also not sure since it's < v1. I need to verify without workspace: first.

**Comment by ghiscoding** [comment]
> That's the change after bumping. Honestly, I'm also not sure since it's < v1. I need to verify without workspace: first.

yeah I was gonna say the same, try without workspace to see what Lerna does (which I'm not sure and I haven't done the code for that), I only worked on adding `workspace:`, the rest was already implemented by Lerna's author long time ago

**Comment by ghiscoding** [comment]
Are you expecting it to bump as well in this use case? 

is it supposed to be the following?
```ts
// from
"workspace:>=0.91.0"

to
"workspace:>=0.92.0"
```
or is it supposed to remain the same as what it was from the start and be left untouched? 
```ts
"workspace:>=0.91.0"
```

**Comment by StarpTech** [comment]
In this concrete example, I'd expect
```diff
---    "SDK": "workspace:>=0.91.0",
+++    "SDK": "workspace:>=0.91.6-next.1",
```
because I only want to bump the pre-version and Lerna can't know if `>=` is no longer appropriate.

**Comment by StarpTech** [comment]
Hi @ghiscoding thank you. I think the best place is to look into the PNPM source. PNPM removes the workspace: protocol after publish.

> There's also [--conventional-prerelease](https://github.com/lerna/lerna/tree/main/commands/version?rgh-link-date=2022-06-06T15%3A18%3A15Z#--conventional-prerelease) that might be useful, I'd probably try the first option I wrote about though

Great tip, I'll check it out.

**Comment by ghiscoding** [comment]
> Hi @ghiscoding thank you. I think the best place is to look into the PNPM source. PNPM removes the workspace: protocol after publish.

I think you mean **during publish**, because "after publish" is pretty much the other issue you opened 😋

**Comment by ghiscoding** [comment]
The fix was pushed under the new release [v1.5.0](https://github.com/ghiscoding/lerna-lite/releases/tag/v1.5.0), thanks for the feedback
